src/js/features/arbitration/reducer.js

```diff
@@ -82,6 +99,44 @@ function reducer(state = DEFAULT_STATE, action) {
         ...state,
         arbitrators: action.arbitrators
       };
+    case BUY_LICENSE:
+      return {
+        ...state,
+        loading: true,
+        error: ''
+      };
+    case BUY_LICENSE_PRE_SUCCESS:
+      return {
+        ...state,
+        txHash: action.txHash
+      };
+    case BUY_LICENSE_SUCCEEDED:
+      return {
+        ...state,
+        licenseOwner: true,
+        loading: false,
+        error: ''
+      };
+    case LOAD_PRICE_SUCCEEDED:
+      return {
+        ...state,
+        price: fromTokenDecimals(action.price, 18)
+      };
+    case CHECK_LICENSE_OWNER:
+      return {
+        ...state, licenseOwner: false
```

Comment:
If we CHECK_LICENSE_OWNER, often, we should not set it to `false`before, because it's gonna cause it to flicker